### ISSUE TITLE
Holy power

### ISSUE THREAD

**Issue by Autobot2000** [ImogenBits/tidyplates#15]
Not sure if it is me or an conflict with another addon but i can not seem to get holy power to show up on nameplates when combo points is checked.

**Comment by ImogenBits** [comment]
Yeah, alternate resources like that still need to be properly implemented. It's on my list.
